noir/noir-repo/compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/generated_acir.rs

```diff
@@ -735,7 +733,8 @@ fn black_box_expected_output_size(name: BlackBoxFunc) -> Option<usize> {
 
         // Output of operations over the embedded curve
         // will be 2 field elements representing the point.
-        BlackBoxFunc::MultiScalarMul | BlackBoxFunc::EmbeddedCurveAdd => Some(2),
+        BlackBoxFunc::MultiScalarMul => Some(2),
```

Comment:
MultiScalarMul can also produce a point at infinite so we should also update this imo.

Reply:
We want to update the msm blackbox function in the stdlib to accept `EmbeddedCurvePoints` as arguments at some point.

Reply:
I added support for point at infinite in MSM.
Let me know if I should also update the inputs to accept `EmbeddedCurvePoints`